pkg/destroy/powervs/vpc-in-cloud-connection.go

```diff
@@ -0,0 +1,209 @@
+package powervs
+
+import (
+	"context"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/pkg/errors"
+	"log"
+	"strings"
+	"time"
+)
+
+// listVPCInCloudConnections removes VPCs attached to CloudConnections and returs a list of jobs.
+func (o *ClusterUninstaller) listVPCInCloudConnections() (cloudResources, error) {
+	var (
+		ctx context.Context
+
+		// https://github.com/IBM-Cloud/power-go-client/blob/v1.0.88/power/models/cloud_connections.go#L20-L25
+		cloudConnections *models.CloudConnections
+
+		// https://github.com/IBM-Cloud/power-go-client/blob/v1.0.88/power/models/cloud_connection.go#L20-L71
+		cloudConnection          *models.CloudConnection
+		cloudConnectionUpdateNew *models.CloudConnection
+
+		// https://github.com/IBM-Cloud/power-go-client/blob/v1.0.88/power/models/job_reference.go#L18-L27
+		jobReference *models.JobReference
+
+		err error
+
+		cloudConnectionID string
+
+		// https://github.com/IBM-Cloud/power-go-client/blob/v1.0.88/power/models/cloud_connection_endpoint_v_p_c.go#L19-L26
+		endpointVpc       *models.CloudConnectionEndpointVPC
+		endpointUpdateVpc models.CloudConnectionEndpointVPC
+
+		// https://github.com/IBM-Cloud/power-go-client/blob/v1.0.88/power/models/cloud_connection_v_p_c.go#L18-L26
+		Vpc *models.CloudConnectionVPC
+
+		// https://github.com/IBM-Cloud/power-go-client/blob/v1.0.88/power/models/cloud_connection_update.go#L20
+		cloudConnectionUpdate models.CloudConnectionUpdate
+
+		foundOne bool = false
+		foundVpc bool = false
+	)
+
+	ctx, _ = o.contextWithTimeout()
+
+	o.Logger.Debugf("Listing VPCs in Cloud Connections")
+
+	select {
+	case <-ctx.Done():
+		o.Logger.Debugf("listVPCInCloudConnections: case <-ctx.Done()")
+		return nil, o.Context.Err() // we're cancelled, abort
+	default:
+	}
+
+	cloudConnections, err = o.cloudConnectionClient.GetAll()
+	if err != nil {
+		log.Fatalf("Failed to list cloud connections: %v", err)
+	}
+
+	result := []cloudResource{}
+	for _, cloudConnection = range cloudConnections.CloudConnections {
```

Comment:
Nit:
```suggestion
	for _, cloudConnection := range cloudConnections.CloudConnections {
```
So there is no need to declare `var cloudConnection` at the beginning of the function.

Reply:
I felt a need.  That is why I went out of my way include not easy to find structure commenting.  This saves me a lot of work in tracking down where in the client is the particular structure definition.